junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/extension/TempDirectoryPerDeclarationTests.java

```diff
@@ -200,7 +200,23 @@ void onlyAttemptsToDeleteUndeletableDirectoriesOnce() {
 		assertSingleFailedTest(results, //
 			instanceOf(IOException.class), //
 			message(it -> it.startsWith("Failed to delete temp directory")), //
```

Comment:
Could you please assert that the actual absolute path of tempdir is listed in the error message?